node/src/main/scala/org/bitcoins/node/models/PeerDAO.scala

```diff
@@ -0,0 +1,76 @@
+package org.bitcoins.node.models
+
+import org.bitcoins.core.p2p.AddrV2Message
+import org.bitcoins.db.{CRUD, SlickUtil}
+import org.bitcoins.node.config.NodeAppConfig
+import slick.lifted.ProvenShape
+
+import java.time.Instant
+import scala.concurrent.{ExecutionContext, Future}
+
+case class PeerDB(
+    address: String,
+    lastSeen: Instant,
```

Comment:
should add `port` as well

Reply:
it also might be easier to store the address as a `ByteVector`, that way we are able to support anything that is in an AddrV2Message